v21.2/local-testing.md

```diff
@@ -8,16 +8,37 @@ This page documents best practices for unit testing applications built on Cockro
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
 cockroach start-single-node --insecure --store=type=mem,size=0.25 --advertise-addr=localhost
 ~~~
 
-Using in-memory storage improves the speed of the cluster for local testing purposes.
+We recommend the following additional [cluster settings](cluster-settings.html) and [SQL statements](sql-statements.html#data-definition-statements) for improved performance during functional unit testing and continuous integration testing:
+
+| Setting                                                      | Value     | Description                                                                                                                                                                          |
+|--------------------------------------------------------------+-----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `kv.raft_log.disable_synchronization_unsafe`                 | `true`    | Improves performance by not syncing data to disk. Data is lost if a node crashes.                                                                                                    |
+| `kv.range_merge.queue_interval`                              | `50ms`    | Frequent CREATE TABLE or DROP TABLE creates extra ranges, which we want to merge more quickly. In real usage, range merges are rate limited because they require rebalancing.        |
+| `jobs.registry.interval.gc`                                  | `30s`     | CockroachDB executes internal queries that scan the jobs table. More schema changes create more jobs, which we can delete faster to make internal job queries faster.                |
+| `jobs.registry.interval.cancel`                              | `180s`    | Timing of an internal task that queries the jobs table. For testing, the default is too fast.                                                                                        |
+| `jobs.retention_time`                                        | `15s`     | More schema changes create more jobs, which affects job query performance. We don’t need to retain jobs during testing and can set a more aggressive delete policy.                  |
+| `schemachanger.backfiller.buffer_increment`                  | `128 KiB` | During table backfills, we fill up buffers which have a large default. A lower setting reduces memory usage.                                                                         |
+| `sql.catalog.unsafe_skip_system_config_trigger`              | `true`    | Turn off gossiping the system config span, which is not necessary in a test environment. (Available in v22.1 and later.)                                                             |
```

Comment:
Let's delete this row for now, wont be available until 22.1

Reply:
Removed it!